docs/source/conf.py

```diff
@@ -15,6 +15,7 @@
 import os
 import sys
 sys.path.insert(0, os.path.abspath('../citrination_client'))
+sys.path.insert(0, os.path.abspath('../..'))
```

Comment:
this is so smelly 😂

Reply:
lol, that answer came from here: https://stackoverflow.com/a/44980548